javascript/ql/src/experimental/Security/CWE-090/LdadInjection.ql

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
You have a typo in the filename for `LdadInjection.ql` (You wrote Lda**d** instead of Lda**p**).

Reply:
Ops, you are right. I made the change. Thanks